php/user's-teams.php

```diff
@@ -0,0 +1,44 @@
+<?php
+///////////////////////////////////////////////////
+/// Return an array of objects with the following
+/// structure:
+/// id: team's id
+/// name: team's name
+/// members: number of team's member
+///////////////////////////////////////////////////
+	require 'connect-db.inc';
+	session_start();
+
+	$db = connect_db();
+	$user_id = $_SESSION ['user_id'];
+
+	$stmt = $db->prepare('SELECT collaborate.team_id, team.name FROM collaborate JOIN team
+							ON collaborate.team_id = team.id WHERE collaborator_id = ?');
+	$stmt->bind_param('i', $user_id);
+	$stmt->execute();
+	$stmt->bind_result($team_id, $team_name);
+
+	/// Store them in an array
+    $teams = array();
+    for ($i = 0; $stmt->fetch(); $i++)
```

Comment:
This will not work because, each iteration you're closing the `$stmt` and preparing a new statement. So split this loop in 2.

The first one will insert `stdClass` objects into the `$teams` array, as you do in line 25 to 27, using the `push_array()`  function. Something like this:
`while ($stmt->fetch()) { $team = new stdClass(); $team->id = ...; array_push($teams, $team); }`

The second one will iterate through the `$teams` array, and for each one it must to execute a prepared statement.
`$stmt = $db->prepare('SELECT ....'); // A statement to get the members of a team`
`foreach ($teams as $team)`
`{`
`    $stmt->bind_param(...);`
`    $stmt->execute();`
`    // bind result and fetch`
`    $team->members = $the_resulf_of_your_query;`
`}`